indexer/data_source.hpp

```diff
@@ -89,6 +89,11 @@ class FrozenDataSource : public DataSource
 
 /// Guard for loading features from particular MWM by demand.
 /// @note This guard is suitable when mwm is loaded.
+/// @note If you need to work with |FeatureType| from different threads you need to use
+/// a unique |FeaturesLoaderGuard| instance for every threads. Construction of
+/// |FeaturesLoaderGuard| should be serialized. Then instances of |FeaturesLoaderGuard|
+/// may be used concurrently. The example of concurrent use of |FeaturesLoaderGuard|
```

Comment:
- you need to use a unique |FeaturesLoaderGuard| instance for every thread
- Then instances of |FeaturesLoaderGuard| may be used concurrently

is there a contradiction here?

Reply:
You're right. I rewrote the comment:
```c++
/// @note If you need to work with FeatureType from different threads you need to use
/// a unique FeaturesLoaderGuard instance for every thread. But construction of instances of
/// FeaturesLoaderGuard should be serialized. For an example of concurrent extracting feature
/// details please see ConcurrentFeatureParsingTest.
```